packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -219,26 +247,52 @@ export const ChatList = ({
               />
             </div>
           )}
-          <div>
-            {chatListState?.chatThreads.map(c => (
-              <Button className={styles.button} key={c.id} onClick={() => onClickChatListItem(c)}>
-                <ChatListItem
-                  key={c.id}
-                  chat={c}
-                  myId={chatListState.userId}
-                  isSelected={c.id === internalSelectedChatId}
-                  isRead={c.id === internalSelectedChatId || c.isRead}
-                />
-              </Button>
-            ))}
-            {chatListState?.moreChatThreadsToLoad === true && (
-              <div className={styles.linkContainer}>
-                <Link onClick={loadMore} href="#" className={styles.loadMore}>
-                  load more
-                </Link>
+          {chatListState && chatListState.chatThreads.length > 0 ? (
+            <>
+              <div>
+                {chatListState?.chatThreads.map(c => (
+                  <Button className={styles.button} key={c.id} onClick={() => onClickChatListItem(c)}>
+                    <ChatListItem
+                      key={c.id}
+                      chat={c}
+                      myId={chatListState.userId}
+                      isSelected={c.id === internalSelectedChatId}
+                      isRead={c.id === internalSelectedChatId || c.isRead}
+                    />
+                  </Button>
+                ))}
+                {chatListState?.moreChatThreadsToLoad === true && (
+                  <div className={styles.linkContainer}>
+                    <Link onClick={loadMore} href="#" className={styles.loadMore}>
+                      load more
+                    </Link>
+                  </div>
+                )}
+              </div>
+            </>
+          ) : (
+            <>
+              <div className={styles.error}>
+                {!chatListState?.userId && <Error message="User not signed-in." subheading={PleaseSignIn}></Error>}
+                {isLoading && (
+                  <div className={styles.spinner}>
+                    <Spinner /> <br />
+                    {chatListState?.status}
+                  </div>
+                )}
+                {chatListState?.status === 'no chats' && (
+                  <Error
+                    icon={LoadingMessagesErrorIcon}
+                    message="No threads were found for this user."
+                    subheading={CreateANewChat}
```

Comment:
In our use case, we already have a way to create a new chat outside I think. If so, is this still needed?

Reply:
In our case, we are directing them to a different page, but this is a contribution to the MGT, so other customers still need a status message.